reactfire/useObservable/useObservable.test.tsx

```diff
@@ -76,6 +77,9 @@ describe('useObservable', () => {
         }
       }
     }
+    /*
+
+    INVESTIGATE not sure what's up with the error component
```

Comment:
This test ensures that errors thrown by hooks can be caught by React's [Error Boundaries](https://reactjs.org/docs/error-boundaries.html)

Reply:
Any idea why it would be failing? Also we need to do the spy cleanup in a after block as it breaks subsequent tests if it fails.

Reply:
Any chance useObservable is swallowing errors?

Reply:
Needed to `catchError` and then reemit only on `.value`